edi_sale_order_import_ubl_endpoint/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "website": "https://github.com/OCA/edi",
     "author": "Camptocamp,Odoo Community Association (OCA)",
     "maintainers": ["simahawk"],
-    "depends": ["edi_sale_order_import", "sale_order_import_ubl", "edi_endpoint_oca"],
-    "auto_install": True,
+    "depends": ["edi_sale_oca", "sale_order_import_ubl", "edi_endpoint_oca"],
+    "auto_install": False,
```

Comment:
The same reason with the comment above

Reply:
I don't see any such information. Could you point me to it?

Reply:
sorry a Thien, my bad, I didn't have a look carefully. Could you update this module like [this commit](https://github.com/thienvh332/edi/pull/28/commits/842dd51f4a1fd7fd71b692a00f58af53063ff762)

And introduce a new module that is similar to this one and make the name `edi_sale_ubl_endpoint_oca`
- Depend on `edi_sale_ubl_oca` and `edi_endpoint_oca`
- Take into account this [xml_id](https://github.com/OCA/edi/blob/e9fe1a70f7db810dfd5da879b8f25a15f490951a/edi_sale_order_import_ubl_endpoint/data/endpoint.xml#L11)